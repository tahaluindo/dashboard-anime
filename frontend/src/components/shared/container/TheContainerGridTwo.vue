<template>
  <section
    class="container relative z-10 grid items-center justify-center grid-cols-1 grid-flow-row px-3 py-10 lg:py-20 mx-auto lg:grid-cols-2 lg:grid-rows-1 gap-10"
  >
    <slot />
  </section>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({});
</script>

<style lang="postcss"></style>
