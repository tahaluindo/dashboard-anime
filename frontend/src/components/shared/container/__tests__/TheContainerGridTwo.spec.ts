import { mount } from "@vue/test-utils";
import TheContainerGridTwo from "../TheContainerGridTwo.vue";

describe("TheContainerGridTwo component test", () => {
  it("should render the default component", async () => {
    const wrapper = mount(TheContainerGridTwo, {
      shallow: true,
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
