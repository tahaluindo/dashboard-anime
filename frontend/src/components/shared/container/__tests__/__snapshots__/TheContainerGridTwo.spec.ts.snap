// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheContainerGridTwo component test should render the default component 1`] = `"<section class=\\"container relative z-10 grid items-center justify-center grid-cols-1 grid-flow-row px-3 py-10 lg:py-20 mx-auto lg:grid-cols-2 lg:grid-rows-1 gap-10\\"></section>"`;
